src/components/NewToDo.js

```diff
@@ -0,0 +1,93 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class NewToDo extends React.Component {
+  state = {
+    title: '',
+    completed: false,
+    userName: '',
+    userNameError: ' \n ',
+    titleError: '',
+  }
+
+  handleSubmit = (event) => {
+    let isWrongData = false;
+
+    event.preventDefault();
+    if (this.state.userName === '') {
+      this.setState({
+        userNameError: 'You must enter User Name',
+      });
+      isWrongData = true;
+    }
+
+    if (this.state.title === '') {
+      this.setState({
+        titleError: 'You must enter what should be done',
+      });
+      isWrongData = true;
+    }
+
+    if (isWrongData) {
+      return;
+    }
```

Comment:
first of all, it's better for clarity code to write one setState per method,
and secondly, if you want to exit the method if `isWrongData` is true, just write return at the end of each of prev conditions since `isWrongData` is always true in those cases